app/assets/js/rtMedia.backbone.js

```diff
@@ -461,7 +461,8 @@ jQuery( function( $ ) {
 			if ( $( '#search_by' ).length > 0 ) {
 				href += '&search_by=' + $( '#search_by' ).val();
 			}
-
+			href = encodeURI( href );
+			console.log( href );
```

Comment:
`console.log( href );` what is this statement is doing here?

Reply:
Removed the line